experimental/web_dashboard/lib/src/pages/item_details.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+import 'package:web_dashboard/src/api/api.dart';
+
+class ItemDetailsPage extends StatelessWidget {
+  final Item item;
+
+  ItemDetailsPage(this.item);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(),
+      body: Center(child: Text('${item.name}'),),
```

Comment:
🤔 I would have expected flutter format to add some line breaks here.

Reply:
good catch. Needed to run `dartfmt -w .`